### ISSUE TITLE
Own potree data only working in Firefox - potree.org pointclouds work in all browsers

### ISSUE THREAD

**Issue by ianh88** [potree/PotreeConverter#341]
Hi,

I couldn't find an answer in the various threads. As the title says I can view the potree pointclouds on any browser on my laptop but when I try to view my own it will only work on Firefox. Ideally I want it to load on Chrome. See below errors.

I have removed the pointcloud file name as it is sensitive.

jquery-3.1.1.min.js:4 Access to XMLHttpRequest at 'file:///D:/XXXX%20Overall%20-%20Potree/libs/potree/resources/lang/dev/translation.json' from origin 'null' has been blocked by CORS policy: Cross origin requests are only supported for protocol schemes: http, data, chrome, chrome-extension, https.

jQuery.Deferred exception: Cannot set property 'innerHTML' of undefined TypeError: Cannot set property 'innerHTML' of undefined
    at initAppearance (file:///D:/XXXX%20Overall%20-%20Potree/libs/potree/potree.js:22484:37)
    at initSidebar (file:///D:/XXXX%20Overall%20-%20Potree/libs/potree/potree.js:22688:2)
    at HTMLDocument.$ (file:///D:/XXXX%20Overall%20-%20Potree/libs/potree/potree.js:19378:5)
    at j (file:///D:/XXXX%20Overall%20-%20Potree/libs/jquery/jquery-3.1.1.min.js:2:29948)
    at k (file:///D:/XXXX%20Overall%20-%20Potree/libs/jquery/jquery-3.1.1.min.js:2:30262) undefined

Anyone any idea whats causing this? Like I mentioned the potree.org stored pointclouds all load fine for me on the same chrome browser.

Thanks,

Ian

**Comment by fwrite** [comment]
In my experience this is related to security measures in the browser that prohibit loading/including local files. In this case it is due to the [CORS](https://en.wikipedia.org/wiki/Cross-origin_resource_sharing) error (indicated in the first line), [which can be disabled in Chrome](https://stackoverflow.com/a/18137280).

If you have python installed, an alternative is to start your own webserver. Go to the root directory of your project and start the server:
python 2: `python -m SimpleHTTPServer 8000`
python 3: `python -m http.server 8000`
Navigate to http://localhost:8000 to see your project. The browser will now handle your website just like the demos.
